src/SFML/Window/OSX/HIDInputManager.mm

```diff
@@ -397,477 +382,805 @@
-        case kHIDUsage_KeyboardStop:                return 0xff;
-        case kHIDUsage_KeyboardAgain:               return 0xff;
-        case kHIDUsage_KeyboardUndo:                return 0xff;
-        case kHIDUsage_KeyboardCut:                 return 0xff;
-        case kHIDUsage_KeyboardCopy:                return 0xff;
-        case kHIDUsage_KeyboardPaste:               return 0xff;
-        case kHIDUsage_KeyboardFind:                return 0xff;
-
-        case kHIDUsage_KeyboardMute:                return 0xff;
-        case kHIDUsage_KeyboardVolumeUp:            return 0xff;
-        case kHIDUsage_KeyboardVolumeDown:          return 0xff;
-
-        case kHIDUsage_KeyboardLockingCapsLock:     return 0xff;
-        case kHIDUsage_KeyboardLockingNumLock:      return 0xff;
-        case kHIDUsage_KeyboardLockingScrollLock:   return 0xff;
-
-        case kHIDUsage_KeypadComma:                 return 0xff;
-        case kHIDUsage_KeypadEqualSignAS400:        return 0xff;
-        case kHIDUsage_KeyboardInternational1:      return 0xff;
-        case kHIDUsage_KeyboardInternational2:      return 0xff;
-        case kHIDUsage_KeyboardInternational3:      return 0xff;
-        case kHIDUsage_KeyboardInternational4:      return 0xff;
-        case kHIDUsage_KeyboardInternational5:      return 0xff;
-        case kHIDUsage_KeyboardInternational6:      return 0xff;
-        case kHIDUsage_KeyboardInternational7:      return 0xff;
-        case kHIDUsage_KeyboardInternational8:      return 0xff;
-        case kHIDUsage_KeyboardInternational9:      return 0xff;
-
-        case kHIDUsage_KeyboardLANG1:               return 0xff;
-        case kHIDUsage_KeyboardLANG2:               return 0xff;
-        case kHIDUsage_KeyboardLANG3:               return 0xff;
-        case kHIDUsage_KeyboardLANG4:               return 0xff;
-        case kHIDUsage_KeyboardLANG5:               return 0xff;
-        case kHIDUsage_KeyboardLANG6:               return 0xff;
-        case kHIDUsage_KeyboardLANG7:               return 0xff;
-        case kHIDUsage_KeyboardLANG8:               return 0xff;
-        case kHIDUsage_KeyboardLANG9:               return 0xff;
-
-        case kHIDUsage_KeyboardAlternateErase:      return 0xff;
-        case kHIDUsage_KeyboardSysReqOrAttention:   return 0xff;
-        case kHIDUsage_KeyboardCancel:              return 0xff;
-        case kHIDUsage_KeyboardClear:               return 0xff;
-        case kHIDUsage_KeyboardPrior:               return 0xff;
-        case kHIDUsage_KeyboardReturn:              return 0xff;
-        case kHIDUsage_KeyboardSeparator:           return 0xff;
-        case kHIDUsage_KeyboardOut:                 return 0xff;
-        case kHIDUsage_KeyboardOper:                return 0xff;
-        case kHIDUsage_KeyboardClearOrAgain:        return 0xff;
-        case kHIDUsage_KeyboardCrSelOrProps:        return 0xff;
-        case kHIDUsage_KeyboardExSel:               return 0xff;
```

Comment:
Might want to keep, but either way works for me

Reply:
To what extent do we care about that? There are seemingly many instances of big switch cases with custom vertical alignment. There is a clang-format option that allows for short case statements on a single line which we might consider using.